backend/compact-connect/lambdas/python/provider-data-v1/handlers/providers.py

```diff
@@ -2,14 +2,48 @@
 
 from aws_lambda_powertools.utilities.typing import LambdaContext
 from cc_common.config import config, logger
+from cc_common.data_model.schema.provider import (
+    GetProviderReadGeneralResponseSchema,
+    QueryProvidersGeneralResponseSchema,
+)
 from cc_common.exceptions import CCInvalidRequestException
-from cc_common.utils import api_handler, authorize_compact
+from cc_common.utils import api_handler, authorize_compact, get_event_scopes
 
 from . import get_provider_information
 
 
+def _user_had_private_read_access_for_provider(compact: str, provider_information: dict, scopes: list[str]) -> bool:
+    if f'{compact}/{compact}.readPrivate' in scopes:
+        logger.debug(
+            'User has readPrivate permission at compact level',
+            compact=compact,
+            provider_id=provider_information['providerId'],
+        )
+        return True
+
+    # we need to check if the user has readPrivate permission at the jurisdiction level
+    # for any of the provider's licenses or privileges
+    relevant_provider_jurisdictions: set = provider_information['privilegeJurisdictions'].copy()
+    relevant_provider_jurisdictions.add(provider_information['licenseJurisdiction'])
```

Comment:
@jusdino If that is the case, I'm not sure how I'm going to determine that from the query endpoint, since we don't have the list of license objects from this endpoint. Is there some other field I should be referencing?

Reply:
it sounds like we need two field to provider records related to licenseJurisdictions 'homeStateLicenseJurisdiction' and 'licenseJurisdictions' which would be a set of all jurisdictions the practitioner has a license in.

Reply:
Confirmed in standup that we will always return the generally available data from the query endpoint, which simplifies things for us.